src/nidmm/metadata/config.py

```diff
@@ -15,10 +15,6 @@
             '64bit': {'name': 'libnidmm.so', 'type': 'cdll'},
         },
     },
-    'context_manager': [
-        {
-            'direction': 'input'
-        },
-    ]
+    'context_manager_name': '_Acquisition',
```

Comment:
This name for the variable is fine for now. I think it will be a little confusing once we add a second context manager to Session for channels.

Reply:
Changed 'context_manager_name' to be a dictionary. This way we can list different context manager names.

I am calling this one 'session'